storage/backend/ceph/internals.go

```diff
@@ -6,14 +6,30 @@ import (
 	"regexp"
 	"strings"
 
+	"encoding/json"
+
 	"github.com/contiv/volplugin/storage"
 
 	log "github.com/Sirupsen/logrus"
 )
 
 func (c *Driver) mapImage(do storage.DriverOptions) (string, error) {
-	blkdev, err := exec.Command("rbd", "map", do.Volume.Name, "--pool", do.Volume.Params["pool"]).Output()
-	device := strings.TrimSpace(string(blkdev))
+	var outputdata map[string]interface{}
+	var device string
+
+	_, err := exec.Command("rbd", "map", do.Volume.Name, "--pool", do.Volume.Params["pool"]).Output()
+	output, err := exec.Command("rbd", "showmapped", "--format", "json").Output()
+
+	json.Unmarshal(output, &outputdata)
+	fmt.Println(outputdata)
+
+	device = ""
```

Comment:
This line is not necessary because it will be the default value of the declared `device` above.